### PR TITLE
testing/fd: more arches

### PR DESCRIPTION
**Pull request by J0WI**

### CODE REVIEW

**Comment by maxice8** [comment]
Is it really required ? i'd rather wait until rust is on all arches as work is being done by mps and Cogitri

**Comment by J0WI** [comment]
It's not required, just nice to have and a way to test the rust ports in real world examples.

File: testing/fd/APKBUILD [hunk 1]
```diff
@@ -2,10 +2,10 @@
 # Maintainer: Leo <thinkabit.ukim@gmail.com>
 pkgname=fd
 pkgver=7.3.0
-pkgrel=0
+pkgrel=1
```
Comment (maxice8) [inline testing/fd/APKBUILD]: no need to bump pkgrel if you're enabling new archs

**Comment by maxice8** [comment]
I'll wait until full rust is available

**Comment by J0WI** [comment]
https://github.com/alpinelinux/aports/commit/cc4e034e2668c4b6a6150d39f3d44200b1562675

**Pull request closed without merging**